docs-developer/call-tree.md

```diff
@@ -61,41 +61,41 @@ With the above graph, we know that the functions at the leaf of the graph were t
+One key point of the aggregation done in the call tree is that it's focused on what _functions_ are called, and the relationships between them. In the profiler we collect frames, that give details about that specific frame of execution. Those are then organized using stacks. Each stack points to a frame, and its prefix (parent) stack. In C++ code a single function can have multiple frames depending on which part of the function was being executed. In JavaScript a function may suddenly be optimized and JITed midway through a series of runs. When the JIT process happens there will be new frames generated for these different implementations of the same function. The stacks in the profile describe the relationship between these individual frames. However, naively using only the frames and stacks will often not produce a particularly useful tree. So when referring to anything in the tree, what we care about is the relationship of functions called, not the individual frames and stacks from the profile. For a detailed explanation of how C++ generates multiple frames for a single function, please read [Frames, funcs, stacks and CallNodes in C++](call-nodes-in-cpp.md).
 
 ## Frames and stacks in JavaScript
 
 Imagine this simplified example of 3 samples of mixed C++, JavaScript Code (js), and JIT optimized JavaScript (JIT). The functions are all labeled as to their implementation.
 
-| Sample index | Sample's stack |
-| ------------ | ---------------|
-| 0 | `JS::RunScript [c++]  ➡  onLoad [js]  ➡  a [js]  ➡  b [js]` |
-| 1 | `JS::RunScript [c++]  ➡  onLoad [js]  ➡  js::jit::IonCannon [c++]  ➡  a [JIT]  ➡  b [JIT]` |
-| 2 | `JS::RunScript [c++]  ➡  onLoad [js]  ➡  js::jit::IonCannon [c++]  ➡  a [JIT]  ➡  b [JIT]` |
+| Sample index | Sample's stack                                                                                 |
+| ------------ | ---------------------------------------------------------------------------------------------- |
+| 0            | `JS::RunScript [c++]  ➡  onLoad [js]  ➡  a [js]  ➡  b [js]`                                 |
```

Comment:
For this bad formatting, I filed an issue on the prettier repository: https://github.com/prettier/prettier/issues/15338

Reply:
Thanks for filing it!